pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -5536,15 +5536,14 @@ status:
 }
 
 func Test_storePipelineSpec(t *testing.T) {
-	labels := map[string]string{"lbl": "value"}
-	annotations := map[string]string{"io.annotation": "value"}
-	pr := &v1beta1.PipelineRun{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:        "foo",
-			Labels:      labels,
-			Annotations: annotations,
-		},
-	}
+	pr := parse.MustParsePipelineRun(t, `
+metadata:
+  name: test-pipeline-run-success
```

Comment:
`name` changed from `foo` to `test-pipeline-run-success` 🤔 it might not have any significance in terms of this test so its fine